tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/TaskProblemAction.java

```diff
@@ -59,9 +60,9 @@ public void run( final TaskProblem taskProblem, final ISelectionProvider provide
     {
         final IFile file = MigrationUtil.getIFileFromTaskProblem( taskProblem );
 
-        new Job( "Marking migration problem as done" )
+        new UIJob( "Marking migration problem as done" )
```

Comment:
can you replace with Job() and then send me the stack trace so I can see what operation is throwing invalid thread exception/